test/e2e/text.cy.js

```diff
@@ -0,0 +1,22 @@
+describe('Text', function () {
+  beforeEach(function () {
+    cy.getData().then(function (data) {
+      this.data = data;
+      cy.visit('/');
+    });
+  });
+
+  it('should display the text component', function () {
+    const textComponent = this.data.components.find((component) => component._id === 'c-15')
```

Comment:
We are not testing against the vanilla course only.
Please find the first text component and navigate to it in a structure and id agnostic way.

Reply:
I didn't know, I was in the process of updating it